README.md

```diff
@@ -115,17 +117,37 @@ foundryup
 curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
 ```
 
+### Installing Colima
+
+```bash
+brew install colima
+colima start
+```
+
+### Installing Docker
+
+```bash
+brew install docker
+```
```

Comment:
These are macOS deps, in Ubuntu you don't need colima, and docker is installed following the [official guide](https://docs.docker.com/engine/install/ubuntu/).

Reply:
[MacOs info](https://github.com/gizatechxyz/avsthon/pull/11/commits/9520572b267b6003a9f47a654fbc80ffb92d12d2)